notebooks/Bohlen_Jacob_get_country.ipynb

```diff
@@ -0,0 +1,1195 @@
+    "        [{\"LOWER\": \"citizens\"}, {\"LOWER\": \"of\"}],\n",
+    "    ]\n",
+    "    \n",
+    "    spans = similar(pattern, doc)\n",
+    "    if spans:\n",
+    "        sentence = str(spans[0].sent)\n",
+    "#         return sentence\n",
+    "        for country in countries:\n",
+    "            if country in sentence:\n",
+    "                return country\n",
+    "    \n",
+    "#         locations: Iterator[str]\n",
+    "#         locations = map(lambda ent: ent.text, self.get_ents(['GPE']))\n",
+    "#         return locations\n",
+    "\n",
+    "#         similar_country: Callable[[str, float], Union[str, None]]\n",
+    "#         similar_country = similar_in_list(countries)\n",
+    "\n",
+    "#         for loc in locations:\n",
+    "#             origin: Union[str, None]\n",
+    "#             origin = similar_country(loc, 0.8)\n",
+    "#             if origin and origin != \"United States\":\n",
+    "#                 return origin"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "a422257a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['Tunisia', 'Ghana', None, 'Honduras', 'Mexico', 'Mexico', None, 'Haiti', 'El Salvador', 'Mexico', 'Honduras', 'El Salvador', 'El Salvador', 'Bangladesh', 'El Salvador', None, 'Brazil', 'Nicaragua', 'India', 'Mexico', None, 'Guatemala', 'Jamaica', 'Guatemala', 'Mali', 'Mexico', None, 'Mexico', 'Guatemala', None, 'El Salvador', 'Guatemala', None, 'Mexico', 'Honduras', 'Iraq', 'Gambia', 'Iraq', 'Nicaragua', 'Mexico', 'Russia', None, 'Eritrea', 'Mexico', None, 'El Salvador', 'Mexico', 'Ethiopia', 'Mexico', 'Nicaragua', None, None, 'Mexico', None, None, 'Ethiopia', 'Guatemala', 'Dominica', None, 'El Salvador', 'Haiti', 'Cameroon', 'Mexico', 'Mexico', 'Mexico', 'Guatemala', 'Mexico', 'Honduras', 'Mexico', None, 'Somalia', None, None, 'Haiti', 'Philippines', 'Honduras', None, 'Honduras', 'Somalia', 'Iraq', 'El Salvador', 'Mexico', None, None, 'China', None, 'Ghana', None, 'Honduras', 'El Salvador', 'Honduras', 'Honduras', None, 'Honduras', 'Mexico', 'Bangladesh', 'Mexico', 'Cameroon', 'Bangladesh', None, None, 'Honduras', 'Haiti', 'Indonesia', 'Gambia', 'Guatemala', None, 'El Salvador', 'El Salvador', 'Mexico', 'Bangladesh', 'South Korea', 'Myanmar', 'Cameroon', 'Guatemala', 'El Salvador', 'El Salvador', 'Dominica', 'Ukraine', 'Nicaragua', None, None, 'El Salvador', 'China', None, 'Pakistan', 'Togo', 'Guatemala', 'El Salvador', None, 'Indonesia', 'Honduras', 'Russia', 'Ethiopia', None, 'Honduras', 'El Salvador', 'Guatemala', 'Mexico', 'Nicaragua', None, 'Venezuela', 'El Salvador', 'Ecuador', 'Ghana', 'El Salvador', None, 'Bangladesh', 'Iraq', 'Honduras', 'Cuba', 'Burkina Faso', 'Cuba', 'Mexico', 'Burkina Faso', 'Tajikistan', 'Cuba', 'Honduras', 'Mexico', 'Liberia', 'China', None, 'Cuba', 'El Salvador', 'Cameroon', 'El Salvador', 'El Salvador', 'China', None, 'Guatemala']\n",
```

Comment:
Not really sure what this is doing, there is approx 135 inputs, and many of them are repeats, including none. The GC already gives us a list of the 216 countries.